pom.xml

```diff
@@ -47,7 +47,7 @@
         <version.kafka>3.3.1</version.kafka>
         <version.org.slf4j>1.7.36</version.org.slf4j>
         <version.logback>1.4.0</version.logback>
-        <version.ybclient>0.8.64-20230929.065833-2</version.ybclient>
+        <version.ybclient>0.8.65-SNAPSHOT</version.ybclient>
```

Comment:
YBClient service is already on `0.8.66` - make sure you increase your version accordingly.